### PR TITLE
SSCS-10834 update hearing venue with gaps venue

### PR DESCRIPTION
**Pull request by pk2453**
https://tools.hmcts.net/jira/browse/SSCS-10834

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: benefit/data/sheets/FixedLists/FixedLists.json [hunk 1]
```diff
@@ -760,160 +760,141 @@
...
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "900001", "ListElement": "Inverness"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "287515", "ListElement": "Ipswich"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "416742", "ListElement": "Newport"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "767032", "ListElement": "Kidderminster"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "671879", "ListElement": "Kings Lynn Mags"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "353615", "ListElement": "Lancaster"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "495952", "ListElement": "Leeds"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "500064", "ListElement": "Leicester"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "195465", "ListElement": "Lincoln"},
+  {"LiveFrom": "23/06/2022", "ID": "FL_hearingEpimsId", "ListElementCode": "345663", "ListElement": "Liverpool 2"},
```
Comment (dattipoeHMCTS) [inline benefit/data/sheets/FixedLists/FixedLists.json:833]: is Liverpool2 valid?

Comment (pk2453) [reply benefit/data/sheets/FixedLists/FixedLists.json:833]: image
Yes

Comment (dattipoeHMCTS) [reply benefit/data/sheets/FixedLists/FixedLists.json:833]: What file are you using?

Comment (dattipoeHMCTS) [reply benefit/data/sheets/FixedLists/FixedLists.json:833]: I don't think we want to be adding inactive venues @pk2453

Comment (pk2453) [reply benefit/data/sheets/FixedLists/FixedLists.json:833]: This is what provided in the Jira. Column U from this sheet. @dattipoeHMCTS

**Review by dattipoeHMCTS — CHANGES_REQUESTED** [review]
Could you please add your version bumps

**Comment by pk2453** [comment]
> Could you please add your version bumps

This is updated

**Review by nickhill111 — APPROVED** [review]

**Review by ikirsanov — DISMISSED** [review]

**Review by dattipoeHMCTS — DISMISSED** [review]

File: benefit/VERSION.yaml [hunk 2]
```diff
@@ -1 +1 @@
-TAG: 5.15.0
+TAG: 5.17.0
```
Comment (bensonlievoco) [inline benefit/VERSION.yaml:1]: I world suggest keep it 5.15.x rather than 17, currently its fine, you have to update it once your pr after po review

**Review by bensonlievoco — DISMISSED** [review]

File: benefit/VERSION.yaml (hunk 2 above)
Comment (pk2453) [reply benefit/VERSION.yaml]: updated

**Review by amoslv1 — APPROVED** [review]

**Review by bensonlievoco — APPROVED** [review]

**Review by dattipoeHMCTS — APPROVED** [review]

**Pull request merged**